packages/data-context/src/sources/migration/legacyOptions.ts

```diff
@@ -86,6 +86,10 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     name: 'exit',
     defaultValue: true,
     canUpdateDuringTestTime: false,
+  }, {
+    name: 'experimentalCspAllowList',
+    defaultValue: false,
+    canUpdateDuringTestTime: true,
```

Comment:
I am thinking we may not want users to update this config during test time as it could lead to inconsistent results between tests.
```suggestion
    canUpdateDuringTestTime: false,
```

Reply:
Since this doesn't actually work due to https://github.com/cypress-io/cypress/issues/21151 we can change this no problem.